Slim/Plugin/Podcast/Parser.pm

```diff
@@ -9,6 +9,8 @@ use Slim::Utils::Cache;
 use Slim::Utils::DateTime;
 use Slim::Utils::Strings qw(cstring);
 
+tie our %recentlyPlayed, 'Tie::Cache::LRU', 50;
```

Comment:
Do you really need this? It's a copy of the pref, but which requires more memory and manual handling to update the pref?...

Reply:
I've moved it to a single file so that it stays a local. The reason for using LRU is really to have recently played episodes sorted automatically. I agree I could use the pref array, find the current item, splice it and re-insert but it seemed to me that a LRU would do the job more easily. I thought that Tie::Cache:LRU was just keeping a hash of scalars/references, but had no copies of actual data, so it was low on memory requirements.